doc/developer-guide/api/functions/TSRemap.en.rst

```diff
@@ -57,12 +58,17 @@ the remap plugin.
 A remap plugin may be invoked for different remap rules. Traffic Server
 will call the entry point each time a plugin is specified in a remap
 rule. When a remap plugin instance is no longer required, Traffic Server
-will call :func:`TSRemapDeleteInstance`.
+will call :func:`TSRemapDeleteInstance`. At that point, it's safe to remove
+any data or continuations associated with that instance.
 
 :func:`TSRemapDoRemap` is called for each HTTP transaction. This is a mandatory
 entry point. In this function, the remap plugin may examine and modify
 the HTTP transaction.
 
+:func:`TSRemapConfigReload` is called once for every plugin when reloading of the
```

Comment:
"just before the remap configuration is reloaded"? Just "when" is a bit ambiguous.

Also, any comment on what happens if the reload fails? Is this called in that case, or not? I would think this would need to be called before the success of the reload can be known.

Reply:
Yes, it has to get called before you know if any of them are going to fail.